base.yaml

```diff
@@ -157,15 +158,30 @@ button:
   - platform: safe_mode
     name: "Safe mode boot"
     entity_category: diagnostic
-  - platform: ratgdo
-    id: ${id_prefix}_sync
-    type: sync
+  
+  - platform: template
+    id: ${id_prefix}_query_status
```

Comment:
```suggestion
    id: ${id_prefix}_query
```

Reply:
I guess we could avoid the breaking change, but I don't think its going to affect most people so may best to no carry it forward

Reply:
Let's do the breaking change. This is too new to worry about it and its a diagnostic entity